clang/include/clang/Basic/arm_sve.td

```diff
@@ -2172,7 +2172,7 @@ let SVETargetGuard = InvalidMode, SMETargetGuard = "sme2" in {
   def SVFCLAMP_X4 : SInst<"svclamp[_single_{d}_x4]",  "44dd",   "hfd",      MergeNone, "aarch64_sve_fclamp_single_x4",  [IsStreaming], []>;
 }
 
-let SVETargetGuard = InvalidMode, SMETargetGuard = "sme2,b16b16"in {
+let SVETargetGuard = InvalidMode, SMETargetGuard = "sme2,sve-b16b16"in {
```

Comment:
It is missing the multi vector targeting Z. For some reason they are not under b16b16.
But they were added in this commit f7392f40f3f6d5d4f.
I can see them being changes in LLVM in this patch
// SME2 Z-targeting non-widening BFloat16 instructions
let Predicates = [HasSME2, HasSVEB16B16] in {

Reply:
Thanks for pointing this out.